pkg/openstacknetconfig/funcs.go

```diff
@@ -118,18 +119,27 @@ func WaitOnIPsCreated(
 	hostStatus *ospdirectorv1beta1.HostStatus,
 ) error {
 	//
-	// If verify that we have the hosts entry on the status if the osnetcfg object
+	// verify that we have the host entry on the status of the osnetcfg object
 	//
 	var osnetcfgHostStatus ospdirectorv1beta1.OpenStackHostStatus
 	var ok bool
 	if osnetcfgHostStatus, ok = osnetcfg.Status.Hosts[hostname]; !ok {
-		cond.Message = fmt.Sprintf("%s %s waiting on node %s to be added to %s config %s",
+		common.LogForObject(
+			r,
+			fmt.Sprintf("%s %s waiting on node %s to be added to %s config %s",
+				obj.GetObjectKind().GroupVersionKind().Kind,
+				obj.GetName(),
+				hostname,
+				osnetcfg.Kind,
+				osnetcfg.Name,
+			),
+			obj,
+		)
+		cond.Message = fmt.Sprintf("%s %s waiting on IPs to be created for all nodes and networks",
```

Comment:
Ah, I see what you mean about keeping the `Message` more uniform across reconciles.  Definitely can envision the old form causing some thrashing.